.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_clone/README.md

```diff
@@ -0,0 +1,18 @@
+# Memory safety proof for aws_cryptosdk_enc_ctx_clone
+
+This proof harness attains 95% code coverage.  The following comments explain
+why the uncovered lines of code are unreachable code.
+
+Some functions contain unreachable blocks of code:
+
+* `aws_hash_table_put`
+
+    * Edge cases never reached. was_created is always false and we never try and put an item in a full table
```

Comment:
IS this a limit of the harness? or of the code itself?

Reply:
The first is of the code itself - was_created is hard-coded as NULL in the aws_cryptosdk_enc_ctx_clone. The second is a property of the Makefile, the MAX_NUM_ELEMS  in a table is 4, well below SIZE_MAX. I have added additional documentation to the README specifying this.